function_lab.js

```diff
@@ -0,0 +1,263 @@
+// Input: `12, 4`
+//
+// Output: `12`
+//
+
+                 //parameter refereing to num1 and num 2
+const largest =(num1, num2) => { // function is searching for the largest varibales
+  if(num1 < num2){ // first requirement num1 < num2
+  return num1 // print result num1
+} if(num1 > num2){ // closing for if with curly brackets // 2req. num1 >num2
+  return num1 // print result num1
+} else if(num1 === num2) { // at last if the 3req. is true
+   return num2 // print num2
+ } // closing with curly brackets the 3 ifstat.
+}
+const resultOfLargest = largest(12,4)
+console.log(resultOfLargest) // print in the value you want the function to use
+// passing 12, 4 as arguments to the largest funtion
+
+
+
+// ## Question 2
+//
+// Write a function named getDogAge that takes one argument: a dog's age in human years. The function returns the dog's age in dog years. The conversion rate is 1 human year to 7 dog years.
+//
+// Example:
+//
+// Input: `4`
+//
+// Output: `28`
+const standardDogYear = 7 //set variable  b/c its 1 humanYear = 7 dogYear
+const getDogAge = (humanYear) => { // placing human years into a function that will search for getDogAge
+ return  humanYear * standardDogYear // print equation  4 * 7
+  }
+  const resultOfTotalDogYear = getDogAge(4)
+console.log(resultOfTotalDogYear); // check varaible getDogAge and use the place holder(humanYear) and add the value 4 to the function
+
+
+// ## Question 3
+//
+// Write a function `arraySum` that takes in an array and returns the sum of all the elements.
+//
+// Example:
+//
+// Input: `[13,2,4,-2]`
+//
+// Output: `17`
+
+// i will need
+// a forloop
+// variable sum that += arr (the index number to )
+
+const arraySum = (array) => { // declared variable in the function as array
+let sum =0 // always start at 0 // asking to start at index o but without the [] b/c its a number we're working with
+for(let i = 0; i < array.length; i++) { //use for loop b/c we're working with an array
+sum += array[i] //equation formolating that
+}
+return sum // the command
+}
+console.log(arraySum([13,2,4,-2]));
+
+//
+// ## Question 4
+//
+// Write a function that takes in any two numbers and return their sum.
+//
+// Example:
+//
+// Input: `14, 5`
+//
+// Output: `19`
+const sumResult =(num5, num6) => { // function and the parameter
+return num5 + num6 // print their sum
+}
+console.log(sumResult(14,5))// the info to evaluate
+
+// ## Question 5
+//
+// Write a function takes in any number grade and returns a corresponding letter grade.
+//
+// | Number Grade | Equivalent Letter Grade |
+// | :--------: | :---------: |
+// | 100 | A+ |
+// | 90 - 99 | A |
+// | 80 - 89 | B |
+// | 70 - 79 | C |
+// | 65 - 69 | D |
+// | Below 65 | F |
+//
+
+const letterGrade = (score) => {
+  if(let score > 90 ) {
+  return 'A';
+} else if (let score > 80) {
+  return 'B';
+} else if(let score > 70) {
+  return 'C';
+} else if (let score > 65) {
+  return 'D';
+} else {
+  return 'F';
+}
+console.log(letterGrade(80));
```

Comment:
On the right track. 

Our syntax isn't correct. We only need let when we are using a for loop. " for (let i = 0;)...
So we have to remove those let's as they cause errors.
Next we are missing a closing bracket.

Another thing is if I run the code with 80 as the score the function returns 'C' when it should return 'B'.
We have to add an equal to sign to our conditions .. Ex: (score >=90)

Last thing, I the score is 100 the function should return A+.. what and where should we add a condition to account for that grade?